tests/integration/src/disaster_recovery_tests.rs

```diff
@@ -418,6 +419,190 @@ fn test_disaster_recovery_flow() {
+    let upgrader_id = system_info.upgrader_id;
+    let admin_user = get_user(&env, WALLET_ADMIN_USER, canister_ids.station);
+
+    // 2. create 3 accounts with the same admin user and no approval required
+    let mut initial_accounts = BTreeMap::new();
+    for account_nr in 0..3 {
+        let create_account_args = AddAccountOperationInput {
+            name: format!("account-{}", account_nr),
+            blockchain: "icp".to_string(),
+            standard: "native".to_string(),
+            read_permission: AllowDTO {
+                auth_scope: station_api::AuthScopeDTO::Restricted,
+                user_groups: vec![],
+                users: vec![admin_user.id.clone()],
+            },
+            configs_permission: AllowDTO {
+                auth_scope: station_api::AuthScopeDTO::Restricted,
+                user_groups: vec![],
+                users: vec![admin_user.id.clone()],
+            },
+            transfer_permission: AllowDTO {
+                auth_scope: station_api::AuthScopeDTO::Restricted,
+                user_groups: vec![],
+                users: vec![admin_user.id.clone()],
+            },
+            transfer_request_policy: Some(RequestPolicyRuleDTO::AutoApproved),
+            configs_request_policy: Some(RequestPolicyRuleDTO::AutoApproved),
+            metadata: vec![],
+        };
+
+        let request = execute_request(
+            &env,
+            WALLET_ADMIN_USER,
+            canister_ids.station,
+            RequestOperationInput::AddAccount(create_account_args),
+        )
+        .expect("Unexpected failed to create account");
+
+        if let RequestOperationDTO::AddAccount(operation) = request.operation {
+            let newly_added_account = operation
+                .account
+                .expect("Unexpected new account not available");
+
+            initial_accounts.insert(
+                newly_added_account.id,
+                (newly_added_account.name, newly_added_account.address),
+            );
+        } else {
+            panic!("Unexpected request operation found");
+        }
+    }
+
+    // 3. perform a reinstall disaster recovery request
+    let init_accounts_input = initial_accounts
+        .iter()
+        .map(|(id, (name, _))| station_api::InitAccountInput {
+            id: Some(id.to_string()),
+            name: name.to_string(),
+            blockchain: "icp".to_string(),
+            standard: "native".to_string(),
+            metadata: vec![],
+        })
+        .collect();
+
+    let res: (ApiResult<()>,) = update_candid_as(
+        &env,
+        upgrader_id,
+        WALLET_ADMIN_USER,
+        "request_disaster_recovery",
+        (upgrader_api::RequestDisasterRecoveryInput {
+            module: station_wasm_module.clone(),
+            arg: Encode!(&station_api::SystemInstall::Init(station_api::SystemInit {
+                name: "Station".to_string(),
+                admins: vec![station_api::AdminInitInput {
```

Comment:
Could we maybe have 3 admins here? Then we should see the quorum being set to 2 as we don't provide any quorum explicitly.

Reply:
Please ignore if this is already tested.